sshmenu/sshmenu.py

```diff
@@ -1,20 +1,27 @@
+import argparse
 import json
 import os
 import readchar
 import sys
 import time
+import shutil
 
 from subprocess import call
 from clint import resources
 from clint.textui import puts, colored
 
 
 def main():
+    # Check arguments
+    parser = argparse.ArgumentParser(prog='sshmenu', description='A simple tool for connecting to remote hosts via ssh.')
+    parser.add_argument('-c', '--configname', default='config.json', help='Specify an alternative configuration name.')
```

Comment:
I think we should make the default value `'config'` and simply append `'.json'` to whatever value comes back. The file extension makes me feel like I can pass in an absolute path, which definitely isn't the case with clint.